gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode,
+  NxP,
+  NxTable,
+  NxTile,
+  NxH2,
+  NxTextLink
+  // NxStatefulAccordion,
+  // NxAccordion
+} from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+// import { GalleryTileFooter } from '../../gallery-components/GalleryTileFooter';
+
+// import NxToastExample from './NxToastExample';
+// import NxToastVariationsExample from './NxToastVariationsExample';
+// import NxToastMultipleExample from './NxToastMultipleExample';
+// // import CodeExample from '../../CodeExample';
+
+// const nxToastExampleCode = require('./NxToastExample?raw'),
+//     nxToastVariationsExampleCode = require('./NxToastVariationsExample?raw'),
+//     nxToastMultipleExampleCode = require('./NxToastMultipleExample?raw');
+//     // nxGlobalHeaderToastExampleCode = require('../../styles/NxGlobalHeader/NxGlobalHeaderToastExample?raw');
+
+const NxToastPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Toasts are rendered on the top right corner of the screen. RSC provides 4 different variations of
+        Toasts: success, warning, info, and error.
+      </NxP>
+      <NxP>
```

Comment:
These docs will need to be updated

Reply:
> These docs will need to be updated

Documentation updated here : [cb8a3e2](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/cb8a3e26b54064eb9a8726948e53ebea5ba5955a)